examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,178 @@
+.. _examples_tools_autotools_autotools_toolchain_build_project_autotools_toolchain:
+
+Build a simple Autotools project using Conan
+============================================
+
+In this example, we are going to create a string formatter application
+that uses one of the most popular C++ libraries: `fmt <https://fmt.dev/latest/index.html/>`_.
+
+We'll use `Autotools <https://www.gnu.org/software/automake/manual/html_node/Autotools-Introduction.html>`_ as build system and `pkg-config <https://www.freedesktop.org/wiki/Software/pkg-config/>`_ as a helper tool in this case, so you should get them installed
+on Linux and Mac before going forward with this example.
```

Comment:
> Are they installed by default in Mac OSX?

No, but they can easily be installed with `brew`, a package manager very popular on OSX.

> Actually those links might not be the best for instructions how to install

Those links are not related how to install, but about the project, in case users want more information.

> Or will be apt install xxxx the best for many users?

I could add section with installation steps on Linux and OSX, but both are not official, as both `brew` and `apt` are not official package managers maintained by Linux or OSX. 

We could explain, "for this example we will use Linux Ubuntu and apt as package manager"

For Mac the same thing, could add `brew` steps.

There are more tools, like `automake` which is not installed by default for both.